test/specs/components/RadioGroup/RadioGroup-test.tsx

```diff
@@ -60,7 +60,13 @@ describe('RadioGroup', () => {
     })
   })
 
-  const itemsTest = getItems => {
+  const itemsTest = ({
+    getItems,
+    isChildrenApiTest = false,
+  }: {
+    getItems: Function
+    isChildrenApiTest?: boolean
+  }) => {
```

Comment:
For now let's leave the two arguments instead of object:

```suggestion
const itemsTest = (getItems: Fuction, isChildrenApiTest: boolean = false) => {
```

One line is better than seven 😸

Reply:
done